hyperspy/io_plugins/edax.py

```diff
@@ -125,6 +125,10 @@ def get_spc_dtype_list(load_all=False, endianess='<'):
     load_all : bool
         Switch to control if all the data is loaded, or if just the
         important pieces of the signal will be read (speeds up loading time)
```

Comment:
It's not much, but it's something. The primary motivation for doing it this way was just to prevent clogging up the metadata structure with a bunch of unimportant values.

On the file I was testing with, the CPU savings is a few milliseconds:

```python
>> %%timeit -n 100
>> s = hs.load('Garnet1.spd', load_all_spc=True)
12.2 ms ± 50.5 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

```python
>> %%timeit -n 100
>> s = hs.load('Garnet1.spd', load_all_spc=False)
9.44 ms ± 26.9 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```